client/src/locale/en.js

```diff
@@ -24,7 +24,7 @@ const en = {
             ["Join", "<p>Invited colleagues who accept the invitation are granted access rights for the applications.</p><br/>", false],
             ["Groups", "<p>The roles are actually group memberships that can be used in SURFconext authorisation rules, or provisioned as attributes or to external SCIM APIs.</p>", false]
         ],
-        footer: "<p>SURFconext Invite offers access management to SURFconext-connected service providers.</p>" +
+        footer: "<p>SURFconext Invite offers access management to SURFconext-connected application providers.</p>" +
```

Comment:
Dit moet denk ik zijn "applications" want we bieden het niet aan aan de leveranciers maar voor de applicaties